tools/lint/lint.xml

```diff
@@ -124,4 +124,7 @@
 
     <!-- DI -->
     <!-- issue id="JvmStaticProvidesInObjectDetector" severity="error" /-->
+
+    <!-- Compose -->
+    <issue id="UnnecessaryComposedModifier" severity="error" />
```

Comment:
A bit (a lot) unrelated, but since I was looking at lint reports, I took the opportunity to also change this, in a dedicated commit.